addons/account/wizard/account_move_send.py

```diff
@@ -0,0 +1,464 @@
+        AND the attachments selected by the user or the one from the template (invoice multi)
+        """
+        self.ensure_one()
+        if self.mode == 'invoice_single':
+            return self._get_selected_attachments_ids() + self._get_linked_attachment_ids(move)
+        else:
+            return mail_template.attachment_ids.ids + self._get_linked_attachment_ids(move)
+
+    def _get_selected_attachments_ids(self):
+        """ Returns all the attachments ids selected by the user (invoice single)
+        """
+        return [vals['id'] for vals in self._get_selected_attachments_data()]
+
+    def _get_selected_attachments_data(self):
+        """ Returns all the attachments data selected by the user (invoice single)
+        """
+        return list(filter(lambda x: not x.get('placeholder'), self.mail_attachments_widget))
+
+    def _get_placeholder_mail_attachments_data(self, move):
+        """ Returns all the placeholder data.
+        Should be extended to add placeholder based on the checkboxes.
+
+        :returns: A list of dictionary for each placeholder.
+        * id:               str: The (fake) id of the attachment, this is needed in rendering in t-key.
+        * name:             str: The name of the attachment.
+        * mimetype:         str: The mimetype of the attachment.
+        * placeholder       bool: should be true to prevent download / deletion
+        """
+        self.ensure_one()
+        filename = self._get_invoice_pdf_report_filename(move)
+        return [{
+            'id': f'placeholder_{filename}',
+            'name': filename,
+            'mimetype': 'application/pdf',
+            'placeholder': True,
+        }]
+
+    def _get_default_mail_attachments_data(self, mail_template, move):
+        """ Returns all the placeholder data and mail template data
+        """
+        results = []
+
+        for attachment in mail_template.attachment_ids:
+            results.append({
+                'id': attachment.id,
+                'name': attachment.name,
+                'mimetype': attachment.mimetype,
+                'placeholder': False,
+            })
+
+        return self._get_placeholder_mail_attachments_data(move) + results
+
+    # -------------------------------------------------------------------------
+    # BUSINESS ACTIONS
+    # -------------------------------------------------------------------------
+
+    @api.model
+    def _get_invoice_pdf_report_filename(self, move):
+        return f"{move.name.replace('/', '_')}.pdf"
+
+    @api.model
+    def _need_document(self, move):
+        return not move.invoice_pdf_report_id and move.state == 'posted'
+
+    @api.model
+    def _download(self, move):
+        """ Download the PDF. """
+        return {
+            'type': 'ir.actions.act_url',
+            'url': f'/web/content/{move.invoice_pdf_report_id.id}?download=true',
+            'close': True, # close the wizard
+        }
```

Comment:
This could be done like this
```python
    def _download(self):
        self.ensure_one()
        return {
            'type': 'ir.actions.act_url',
            'url':  '/web/content/account.move/%s/invoice_pdf_report_id' % self.id,
            'close': True, # close the wizard,
        }
```